src/modules/user/routes/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const router = express.Router();
+const signUpRoute = require('./signUp')
+const loginRoute = require('./login')
+const verifyUserRoute = require('./verifyUser')
+const refreshTokenRoute = require('./refreshTokenRoute')
+const resetPassword = require('./resetPassword');
+const logoutUser = require('./logout');
+const updateProfile = require('./updateProfile');
+const viewProfile = require('./viewProfile');
+
+
+router.use('/user', signUpRoute)
+router.use('/user', loginRoute)
+router.use('/user', logoutUser)
+router.use('/user', updateProfile)
+router.use('/user', viewProfile)
+router.use('/user/verify', verifyUserRoute)
+router.use('/user/refresh', refreshTokenRoute)
+router.use('/user/reset', resetPassword)
+
```

Comment:
By this method you can add `N` no of routes with importing them.

```suggestion
const express = require('express');
const router = express.Router();
const path = require('path');
const glob = require('glob');

glob.sync(path.join(__dirname, './*.js')).forEach((file) => {
    if (file !== __filename) {
        router.use(require(file));
    }
});

```